elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -478,6 +525,20 @@ def grade(element_html, data):
         num_initial_correct, true_answer_length = grade_dag(submission, depends_graph, group_belonging)
         first_wrong = -1 if num_initial_correct == len(submission) else num_initial_correct
 
+        if feedback_type == 'first-wrong':
+            for i, block in enumerate(student_answer):
+                if i >= num_initial_correct: break
+                block["badge_type"] = "badge-success"
+                block["icon"] = "fa-check"
+
+            if first_wrong != -1:
+                student_answer[first_wrong]["badge_type"] = "badge-danger"
+                student_answer[first_wrong]["icon"] = "fa-xmark"
+
+                for block in student_answer[first_wrong + 1: ]:
+                    block["badge_type"] = ""
+                    block["icon"] = ""
```

Comment:
shouldn't this for loop be indented 1 less time than it is?

Reply:
It could be, but if `first_wrong` is -1then doesn't that mean that the solution so far is correct? The body of this if statement adds an X badge to the incorrect block and then populates any remaining blocks with blank badges.

Reply:
ah, my bad I thought that this for loop was under the other for loop. Angry face at python's indentation-based scoping.